go/chat/helper.go

```diff
@@ -97,24 +97,14 @@ func (s *sendHelper) conversation(ctx context.Context) error {
 		return err
 	}
 	uid := gregor1.UID(kuid.ToBytes())
-
-	oneChatPerTLF := true
-	convs, _, err := FindConversations(ctx, s.G(), s.DebugLabeler, s.remoteInterface, uid, s.canonicalName,
-		chat1.TopicType_CHAT, s.membersType, chat1.TLFVisibility_PRIVATE, s.topicName, &oneChatPerTLF)
+	conv, _, err := NewConversation(ctx, s.G(), uid, s.canonicalName, s.topicName,
```

Comment:
is there a usecase for sending many messages to the same channel? I figured it would be bad to call `NewConversation` every single time